### ISSUE TITLE
Unable to authenticate with the Rancher API after new v1.4.0 release

### ISSUE THREAD

**Issue by ishantanu** [rancher/terraform-provider-rancher2#94]
Hi,

I am not able to authenticate with the Rancher API after the latest release.

```
Error: Bad response statusCode [401]. Status [401 Unauthorized]. Body: [message=must authenticate] from [https://xyz.rancher.com/v3]
```

The TF code is same as it was previously with v1.3.0 and even when I built my own plugin binary from this repository. The latest commit at the time of building it was:
```
commit a37273a8923ce5b6c0916f62699ae94b8b66cef9 (HEAD -> master)
Merge: ea12d08 203e4cf
Author: Raúl Sánchez <rawmind0@users.noreply.github.com>
Date:   Sun Aug 4 20:53:43 2019 +0200

    Merge pull request #76 from rawmind0/customca

    Added custom_ca argument on etcd s3_backup_config on rancher2_cluster and rancher2_etcd_backup resources
```

Clearly, something has changed in the latest release which is not allowing me to authenticate with the Rancher API.

The TF code which was working with earlier release is:

```
resource "random_string" "rancher_cloud_cred_random" {
  count            = var.existing_vpc ? 1 : 0
  upper            = false
  length           = 8
  special          = false
  override_special = "/@\" "
}

# Provider config
provider "rancher2" {
  api_url    = var.rancher_api_url
  access_key = var.rancher_access_key
  secret_key = var.rancher_secret_key
}

resource "rancher2_cloud_credential" "test" {
  count            = var.existing_vpc ? 1 : 0
  name             = "cc-${random_string.rancher_cloud_cred_random[count.index].result}"

  amazonec2_credential_config {
    access_key = var.AWS_ACCESS_KEY_ID
    secret_key = var.AWS_SECRET_ACCESS_KEY
  }
}
```

This code works just fine if I manually copy my locally built binary instead of the latest release.

**Comment by ChrisMcKee** [comment]
I just set the version to 1.3 for now
```
provider "rancher2" {
  api_url    = "https://xxxxx/v3"
  access_key = "${var.rancher2_access_key}"
  secret_key = "${var.rancher2_secret_key}"
  version = "<= 1.3.0"
}
```

**Comment by ishantanu** [comment]
Or you can use `token_key` instead of `access_key` and `secret_key`. `token_key` is `rancher2_access_key:rancher2_secret_key` in simple terms. Or just copy it from the Rancher API keys UI. This works.

**Comment by rawmind0** [comment]
@ishantanu , thanks for reporting this. As you mentioned, `token_key` is working fine.

Submitted PR #95 to address it.

**Comment by rawmind0** [comment]
Released v1.4.1 that fix auth issue.
